engine/apps/api/views/shift_swap.py

```diff
@@ -33,13 +33,6 @@ class BaseShiftSwapViewSet(ModelViewSet):
     serializer_class = ShiftSwapRequestSerializer
     pagination_class = FiftyPageSizePaginator
 
-    def _do_create(self, beneficiary: User, serializer: BaseSerializer[ShiftSwapRequest]) -> None:
-        shift_swap_request = serializer.save(beneficiary=beneficiary)
-
-        write_resource_insight_log(instance=shift_swap_request, author=self.request.user, event=EntityEvent.CREATED)
-
-        create_shift_swap_request_message.apply_async((shift_swap_request.pk,))
-
```

Comment:
since both the internal/public API views + now the Google Calendar celery task all have this logic of creating the `ShiftSwapRequest` object + submitting a `create_shift_swap_request_message` task, I decided to move this logic to a `ShiftSwapRequest` `post_save` django signal ([here](https://github.com/grafana/oncall/pull/4104/files#diff-f7ab2fa3cbeb4b36eab0b7904b2c0dd89a780cd73421d32490526538eb1ca68dR264-R272))